packages/app/public/static/locales/en_US/admin.json

```
@@ -1043,6 +1043,7 @@
     "paste_transfer_key": "Paste transter key here"
   },
   "g2g": {
+    "transfer_success": "G2G transfer successfully completed",
```

```suggestion
    "transfer_success": "Completed GROWI to GROWI transfer successfully",
```
UI に `GROWI to GROWI transfer` と表示しているのと、`successfully` は最後に来るイメージ。